src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h

```diff
@@ -4,6 +4,14 @@
 #include "ultra64.h"
 #include "global.h"
 
+typedef enum {
+    /* 0 */ PAUSE_BG_PRERENDER_OFF, // Inactive, do nothing.
```

Comment:
this header is imo for things internal to the overlay, but this enum is used outside the the kaleido overlay